### DISCUSSION TITLE
学位论文类型的条目信息如何添加导师栏

### DISCUSSION THREAD

**Discussion by TJYYWXBG** [zotero-chinese/wiki#248] [求助 / Q&A] [answered]
![2024-08-04_165802](https://github.com/user-attachments/assets/fb02d386-8423-442d-b55d-500a50763d60)
如图，默认是没有导师这一项信息，该怎么添加显示出来啊

**Comment by zepinglee** [comment, marked as answer]
Zotero 的数据类型没有学位论文的“导师”，不过一般的参考文献体例也不会显示“导师”，所以总的问题不大。建议将导师信息录入 `contributor` 类型。

另外论文类型需要填写完整的“硕士学位论文”，建议更新知网对应的翻译器 translator。

**Reply by TJYYWXBG** [reply]
谢谢明白了，作者，贡献者，感觉还可以再添加个导师或者自定义
【另外论文类型需要填写完整的“硕士学位论文”，建议更新知网对应的翻译器 translator。】
这个不太明白，不过翻译器刚才确实只能导入条目，不能下载论文pdf
